app/models/katello/concerns/smart_proxy_extensions.rb

```diff
@@ -59,7 +59,7 @@ def refresh
         validates :download_policy, inclusion: {
           :in => DOWNLOAD_POLICIES,
           :message => _("must be one of the following: %s") % DOWNLOAD_POLICIES.join(', ')
-        }
+        }, :if => lambda { |sp| sp.download_policy }
```

Comment:
Is this just a nil check?

Reply:
yep!  so :if => :download_policy should be sufficient!